package.json

```diff
@@ -17,9 +18,10 @@
     "qs": "^6.5.2",
     "vue": "^2.5.16",
     "vue-async-computed": "^3.4.1",
-    "vuetify": "^1.1.9"
+    "vuetify": "^1.3.0"
```

Comment:
This change doesn't have any effect, does it?

Reply:
I think I accidentally upgraded Vuetify somehow, but I think we probably should update Vuetify in a probably PR for its fixes and new features.

Reply:
Agreed, since the upgrade of Vuetify is out of scope of this PR, can you also back out those changes from the yarn.lock? We can add it on a second PR and then reviewers will know to poke around the components to make sure everything looks nice.

Reply:
Great catch. I rebased again to try to clean up the yarn.lock.